tests/unit/test_google_analytics.py

```diff
@@ -0,0 +1,62 @@
+from aiohttp.test_utils import make_mocked_request
+
+from app.google_analytics import ga_ua_id_processor
+from . import RHTestCase
```

Comment:
```suggestion
from tests.unit import RHTestCase
```

We should stick to absolute imports instead of relative.